src/main/java/org/opentripplanner/routing/edgetype/StreetTraversalPermission.java

```diff
@@ -20,11 +18,12 @@
     BICYCLE_AND_CAR(4 | 2),
     ALL(4 | 2 | 1);
 
-    private static final Map<Integer, StreetTraversalPermission> lookup = new HashMap<Integer, StreetTraversalPermission>();
+    private static final StreetTraversalPermission[] lookup = new StreetTraversalPermission[8];
```

Comment:
Can you replace the `8` with a call to `EnumSet.allOf(StreetTraversalPermission.class).size()` in the static initializer?

Reply:
Done